addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -2304,6 +2304,76 @@ var FieldRadio = FieldSelection.extend({
     },
 });
 
+
+var FieldSelectionBadge = FieldSelection.extend({
+    template: null,
+    className: 'o_field_selection_badge',
+    tagName: 'span',
+    specialData: "_fetchSpecialMany2ones",
+    events: _.extend({}, AbstractField.prototype.events, {
+        'click span.o_selection_badge': '_onBadgeClicked',
+    }),
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @override
```

Comment:
Linting: shouldn't it be a bit more useful ? Either we don't add unnecessary lines, either we really use it ( cc @ged-odoo am I right ?)

Reply:
this is fine I think.  The goal is to make people aware that this is a private function (even though it is evident, this makes it more concrete) and the fact that this is an override is quite useful.  This means that the API should not be changed (without care)